### PR TITLE
Pull request for issue #703

### LINKED ISSUE #703: 🐛 [Design Fix]: The line under the text hide the text
### Descrcibe the bug?

The underlined line hides the text  as you can see it in the attached screen shot
image

### Steps to reproduce the bug?

Go on to the home page of the reactpaly.io and scroll down to the What our community says section

### Expected behavior

The underline should be behind the text as shown in the attached screen shot.

image

This behavior is already implement in the footer as you can see in the attached screen shot.
image

### Desktop (Please provide your system information)

OS:Windows
browser: Chrome

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

Current behavior
image

Expected behavior
image

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Hat52**
# Description

The underline was hiding the text behind. Now it is moved to the back of the text. closes #703 

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

I have tested the changes by running them on my local.

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by atapas — COMMENTED** [review]
@Hat52 Check the diff.

There are so many diffs. I am not sure if all are intended.

**Comment by Hat52** [comment]
@atapas I think these difference are because of the pretter extension that I have installed in my vs code. It formatted the file.

File: src/common/playcreated/play-created.scss [hunk 1]
```diff
@@ -1,88 +1,88 @@
...
+	font-size: var(--fs-xl);
+	text-align: center;
+	color: var(--color-neutral-80);
+	strong {
+		font-size: var(--fs-xxl);
+		::before {
+			content: '';
+			position: absolute;
+			height: 4px;
+			// background-color: var(--color-brand-primary);
```
Comment (koustov) [inline src/common/playcreated/play-created.scss:24]: remove commented code

**Review by koustov — CHANGES_REQUESTED** [review]
a tiny comment

**Comment by koustov** [comment]
> @atapas I think these difference are because of the pretter extension that I have installed in my vs code. It formatted the file.

I can see the diffs are beyond prettier changes. Lots of changes around motivation play. Not sure whats going on.

**Comment by Hat52** [comment]
> > @atapas I think these difference are because of the pretter extension that I have installed in my vs code. It formatted the file.
> 
> I can see the diffs are beyond prettier changes. Lots of changes around motivation play. Not sure whats going on.

**Pull request closed without merging**